spec/tic_tac_toe_spec.rb

```diff
@@ -1,7 +1,49 @@
+require 'game'
+require 'output'
+require 'input'
+
+
 RSpec.describe TicTacToe do
 
-  it "asserts true equsls true" do
-    expect(true). to eq(true)
+  before(:each) do
+    @game = Game.new
+    @output = Output.new
+    @test_input = Input.new(["7", "8"])
+    @app = TicTacToe::Welcome.new(@game, @output, @test_input)
+
   end
-  
+
+    it "should expect welcome to have been called" do
+      expect(@app).to receive(:welcome).with(no_args)
+      @app.welcome
+    end
+
+    it "should prints out draw message when the game is a draw" do
+      @game.set_marks(
+        ['X', 'O', 'X'],
+        ['X', 'X', 'O'],
+        ['O', 'X', 'O'])
+
+      expect{@app.start_game}.to output(include(@output.draw_text)).to_stdout
```

Comment:
Then you won't have to use this {}.to output() pattern.  Instead, you can just check the Mock that the right output methods were called.